docs/contributor/contribute.rst

```diff
@@ -0,0 +1,6 @@
+Contributing to a lesson
+=========================
+
+Thank you for improving Oppia's lessons with your contributions! This section will cover how you can contribute:
```

Comment:
Should we assume that they are interested in contribution OR that have already done the contribution?

If the first one then I think we should go with "Thanks for your interest in contributions in Oppia lessons." (or something around that maybe?

Reply:
Yes good point, I changed it now. Done